packages/integrations/cloudflare/src/server.directory.ts

```diff
@@ -10,25 +10,16 @@ if (!isNode) {
 export function createExports(manifest: SSRManifest) {
 	const app = new App(manifest);
 
-	const onRequest = async ({
-		request,
-		next,
-		...runtimeEnv
-	}: {
-		request: Request & CFRequest;
-		next: (request: Request) => void;
-		waitUntil: EventContext<unknown, any, unknown>['waitUntil'];
-	} & Record<string, unknown>) => {
-		process.env = runtimeEnv.env as any;
+	const onRequest = async (context: EventContext<unknown, string, unknown>) => {
+		const request = context.request as CFRequest & Request
+		const { next, env } = context
+
+		process.env = env as any; // would love to remove this any cast in the future
```

Comment:
It would be beneficial to add a comment that explains **why** the cast to `any`, like this your comment would give some context to other developers.